bundles/Planning/Resources/assets/planning.js

```diff
@@ -0,0 +1,304 @@
+import Vue from 'vue';
+import axios from 'axios';
+
+require('./planning.css');
+
+(function () {
+    const currentUrl = window.location.href;
+
+    const app = new Vue({
+        el: '#app',
+        data: {
+            hideUsers: {},
+            toggle: {},
+            sprints: [],
+            users: {},
+            projects: {},
+            numberLoaded: 0,
+            projectFilter: '',
+            userFilter: ''
+        },
+        computed: {
+            sortedUsers: function () {
```

Comment:
A comment on how were sorting would be nice, e.g. “Sort users by displayName”

Reply:
done